core/dispatcher/src/main/scala/whisk/core/dispatcher/Dispatcher.scala

```diff
@@ -191,8 +195,6 @@ object Dispatcher extends Logging {
     def requiredProperties =
         Map(servicePort -> 8080.toString()) ++ kafkaHost
 
-    val executionContext = ExecutionContextFactory.makeCachedThreadPoolExecutionContext()
```

Comment:
Did we have any evidence that this is superior to the default? @rabbah @perryibm